decidim-admin/config/locales/ca.yml

```diff
@@ -133,9 +133,6 @@ ca:
         organization_url: URL de l'organització
         redirect_uri: URL de redirecció
         site: Lloc web
-  date:
```

Comment:
You shouldn't change locales in other languages, as Crowdin will automatically do it when the PR is merged. Otherwise you risk having conflicts (which looks like already happened).